docs/table/performance.inc.rst

```diff
@@ -39,3 +39,39 @@ then use ``serialize_method='data_mask'``.
 It uses the non-masked version of data and it is faster::
 
     >>> tm.write('tm.ecsv', overwrite=True, serialize_method='data_mask') # doctest: +SKIP
+
+Read FITS with memmap=True
+--------------------------
+
+By default :meth:`~astropy.table.Table.read` will read the whole table into memory, which 
+can take a lot of memory and can take a lot of time, depending on the table size and 
+file format. In some cases, it's possibly to only read a subset of the table by choosing 
```

Comment:
```suggestion
file format. In some cases, it's possible to only read a subset of the table by choosing 
```

Reply:
It might be useful to clarify the "In some cases". I *think* that the table will always be loaded fully in memory if one column must be converted to another format, i.e. when there is a boolean column, or when some column must be scaled. 
This is if I remember correctly the discussions on @astrofrog 's PR, so maybe he remembers better than me ;).